packages/react-compass/src/checkbox/checkbox.styles.ts

```diff
@@ -116,21 +113,19 @@ export const StyledCheckboxCheckmark = styled('span', {
   },
 })
 
-export const StyledCheckboxLabelContent = styled('span', {
-  marginLeft: '$7', // checkbox area + padding
-})
-
 export const StyledCheckboxLabel = styled('label', {
   position: 'relative',
   fontSize: '$label1',
   cursor: 'pointer',
   userSelect: 'none',
   borderRadius: '3px',
-  display: 'inline-block',
   whiteSpace: 'nowrap',
   minHeight: '$5',
   minWidth: '$5',
   padding: '2px',
+  display: 'inline-flex',
+  alignItems: 'center',
```

Comment:
I think should be move to h5 variant to avoid affecting the old component

Reply:
@hoapn-fsoft I did refactor the checkbox a little bit, from using absolute position in the checkbox to using flexbox, making sure the checkbox and the label are aligned horizontally.

Reply:
ok, double check it with some cases like in the table,...

Reply:
![image](https://github.com/comfortdelgro/compass-design/assets/7071853/73c4cf67-3e21-4562-a0c5-36c85cbe13d5)
![image](https://github.com/comfortdelgro/compass-design/assets/7071853/258eddae-6b63-4f4f-8881-df14fc85a995)

double checked on table v1 and dialog @hoapn-fsoft

Reply:
thanks @pieliedie , resolved